scss/core/abstracts/_variables.scss

```diff
@@ -75,9 +75,9 @@ $body-color:                $gray-600 !default;
 //
 // Style anchor elements.
 
-$link-color:                theme-color("primary") !default;
+$link-color:                $blue-500 !default;
 $link-decoration:           none !default;
-$link-hover-color:          darken($link-color, 15%) !default;
+$link-hover-color:          $blue-500 !default;
```

Comment:
@Pallavi-PH  -- isn't it a good idea to change the color a little bit on hover over the link?